headset-charge-indicator.py

```diff
@@ -230,8 +241,7 @@ def switch_menu():
 def refresh(switch_command_is_present: bool = False):
     change_label(None)
     change_chatmix(None)
-    if switch_command_is_present:
-        change_icon(None)
+    change_icon(None)
```

Comment:
If we do not use "switch_command_is_present" any more we could remove it and simplify the call to refresh() above.

Reply:
Adding this in causes an issue for me where the icon is not properly set until the first refresh. Without this change, the icon is properly set on start.